src/directives/CommonDirective.js

```diff
@@ -0,0 +1,63 @@
+const CommonDirective = {
+  bind(el, binding, vnode) {
+    const { context } = vnode;
+    const { $attrs } = context;
+    // backgroundColor, width, height, top, left, flexShrink, flexGrow, order.
+    if (Object.keys($attrs).length) {
+      let cssString = '';
+      cssString = $attrs.backgroundColor
+        ? cssString
```

Comment:
es6 templates are perfect for this. so many concats here.

`cssString += $attrs.backgroundColor ? `background-color:${$attrs.backgroundColor};` : '';`